src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import {Routes,Route} from "react-router";
+import  {Home} from './pages/Home/Home';
+import {MonthStatistics} from "./pages/MonthStatistics/components/MonthStatistics";
+import {Header} from "./pages/shared/Header/Header";
+import {Container} from "./styles/styles"
+
+function App() {
+  return (
+    <Container>
```

Comment:
You can define own routes as children? use RRD v6 
```suggestion
const routes = [
     {
      path: "/",
      element: <Container />,
      children: [
        {
          path: "home",
          element: <Home />,
        },
        { 
           path: "statistics",
           element: <MonthStatistics /> 
         },
      ],
    },
]
```